Tribler/Test/Core/Modules/RestApi/test_debug_endpoint.py

```diff
@@ -53,7 +50,7 @@ def verify_response(response):
         self.should_check_equality = False
         return self.do_request('debug/circuits', expected_code=200).addCallback(verify_response)
 
-    @deferred(timeout=10)
+    #@deferred(timeout=10)
```

Comment:
Please just remove these if they are not needed (holds for the rest of the file as well).

Reply:
I left these in place because they hold information on test timeouts. Most probably this holds no value, so I'll remove it.

Reply:
In general we have the practice to not leave commented out code in the codebase. We have Git in case the code turned out to be important and we need it back.